### ISSUE TITLE
Help to use IOU metric

### ISSUE THREAD

**Issue by RaminNateghi** [bubbliiiing/efficientdet-keras#6]
Hi, 
I want to use a metric like IOU for training. How can I define an IOU metric for training?
Thanks.

**Comment by bubbliiiing** [comment]
I don’t understand what IOU metric is.
The object detection I understand uses map as an evaluation metric

**Comment by RaminNateghi** [comment]
Yes, in addition to the MAP we can use intersection-over-union metric for object detection as well. The MAP is also ok, how can I calculate MAP as an evaluation metric during training?

**Comment by bubbliiiing** [comment]
Sorry, I didn't write the map calculation code during training. During training, only LOSS is used as an evaluation indicator. This code is difficult to write, and I don’t have time to finish it yet
